jest.config.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+    collectCoverage: true,
+    collectCoverageFrom: [
+        'src/**',
+    ],
+    verbose: true,
+    coverageDirectory: '.coverage',
+    testMatch: [
+        '<rootDir>/spec/**',
+    ],
+    transformIgnorePatterns: ['<rootDir>/node_modules/'],
+};
```

Comment:
@jairo-bc Maybe let's add the "coverageThreshold" equal to the current level of test coverage, so that we don't forget to add tests for the new code in the future

Reply:
I think first we need to add those tests to match threshold. We can have a low threshold for now and increase that in future.

Reply:
Yeah, I meant at the moment we have 30..36% here, so we can just add smth like:
```
coverageThreshold: {
    global: {
        branches: 30,
        functions: 36,
        lines: 35,
        statements: 36,
    },
}
```
or less strict:
```
coverageThreshold: {
    global: {
        branches: 30,
        functions: 30,
        lines: 30,
        statements: 30,
    },
}
```

But it's optional :)

Reply:
Also, it makes sense to create a JIRA ticket for increasing the test coverage for this library if we are gonna support it in the future.
@jairo-bc @junedkazi

Reply:
Jira ticket is here https://jira.bigcommerce.com/browse/STRF-8658